tests/framework/AnalyticModels/optimizing/maljovec_precond.py

```diff
@@ -0,0 +1,5 @@
+def evaluate(y):
```

Comment:
I like it named specifically as the preconditioner for a particular function, or not as a preconditioner at all if it has broader application.  I'll change it.